src/python/pants/backend/python/packaging/pyoxidizer/rules_integration_test.py

```diff
@@ -1,13 +1,22 @@
 # Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
+import platform
 import subprocess
 from pathlib import Path
 from textwrap import dedent
 
+import pytest
+
 from pants.testutil.pants_integration_test import run_pants, setup_tmpdir
 
+skip_on_linux_arm = pytest.mark.skipif(
+    platform.system() == "Linux" and platform.machine() == "aarch64",
+    reason="PyOxidizer is not supported on Linux ARM",
```

Comment:
That seems surprising.

Reply:
I was surprised too, but https://pyoxidizer.readthedocs.io/en/pyoxidizer-0.17/pyoxidizer_getting_started.html:

```
PyOxidizer is officially supported on the following operating systems:

- Windows x86 (32-bit)
- Windows x86_64/amd64 (64-bit)
- macOS x86_64 (Intel processors)
- macOS aarch64 (ARM/Apple processors)
- Linux i686 (32-bit)
- Linux x86_64 (64-bit)
```